### ISSUE TITLE
Move Accessibility guidance on PW out of the "QA and Accessibility" parent page

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#49529]
## User Story
As a VFS team member, I want to be able to find Accessibility guidance in as few clicks as possible so that I can access the information quickly and efficiently.

Assignee: @briandeconinck 
Peer Reviewer: @CherylEvans 

## Description
Currently, the Accessibility guidance on Platform Website/Developer docs is bundled with the QA guidance. This was likely because both practice areas were managed by the former QA Standards team. The bundling doesn't make sense, and ideally both QA and Accessibility should have their own parent headings under Developer Docs.

## Impacted Artifacts
[QA and accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/qa-and-accessibility-testing) parent page
[Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-testing) page

## Tasks
- [x] Move Accessibility testing page out of the QA and Accessibility testing parent page
   - You may need to check with Platform Content since we're moving a whole set of pages
- [x] Rename "Accessibility testing" page "Accessibility" - @briandeconinck this is your call
- [x] Rename "QA and accessibility testing" to "Quality Assurance (QA)"

## Peer Review
_To be completed by peer reviewer_
- [x] User story and acceptance criteria are met

## Acceptance Criteria
- [x] Accessibility guidance is no longer nested under QA

### Refinement
- [x] Ticket has user story, description, tasks, and acceptance criteria
- [x] Ticket has [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization), if applicable
- [x] Ticket has been sized 
- [x] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [x] Ticket has been discussed as a team at Planning
- [x] Ticket has been assigned to appropriate initiative or quarterly epic
- [x] Ticket has been assigned to a Sprint
- [x] Ticket has been moved to the Planned pipeline in the Governance team board
- [x] Ticket as an assignee and peer reviewer

**Comment by briandeconinck** [comment]
@shiragoodman I'm looking at this today with some fresh eyes and having some second thoughts.

Everything under the [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-testing), with thoughts...

Child pages:
- ~Accessibility standards~ -- Archived in #49312
- ~WCAG 2.1 success criteria and foundational testing~ -- Archived in #47916
- [Add accessibility test cases in TestRail](https://depo-platform-documentation.scrollhelp.site/developer-docs/add-accessibility-test-cases-in-testrail)
   - This is definitely a QA/developer resource, and makes sense to live somewhere in the Developer Docs world
- [Accessibility defect severity rubric](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric)
   - It's not totally clear to me who the intended audience is for this page. I know I refer to it sometimes when I'm trying to decide on the severity level for an issue. I could imagine a PM looking at it to try to understand the differences between severity levels, but I don't know that they would care about all of the details included here. But I don't know that it makes sense as a *developer doc* specifically.
   - My gut reaction here would be to suggest this get converted to an internal guidance, with a simplified version included on the ["Prepare for"](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review) page in the collab cycle section.
- [Request support from the VA 508 office](https://depo-platform-documentation.scrollhelp.site/developer-docs/request-support-from-the-va-508-office)
   - This is definitely not a developer doc, and is most likely relevant to PMs. I think if it fits anywhere it probably fits in with the collab cycle since we point people to the audit request section after staging. But the document review section doesn't quite fit with that either.
- [Daily accessibility scan for VA.gov static content](https://depo-platform-documentation.scrollhelp.site/developer-docs/daily-accessibility-scan-for-va-gov-static-content)
   - I'm not sure who the audience is for this one either, since it's not really relevant to VFS teams. Probably the only people who care are Platform and Sitewide. It might make sense as internal documentation.

Other stuff linked from that page:
- [End-to-end testing with Cypress](https://depo-platform-documentation.scrollhelp.site/developer-docs/end-to-end-testing-with-cypress)
   - Info about e2e and Cypress, not explicitly about accessibility. Currently lives under Developer Docs and that makes sense.
- [Keyboard testing helper functions](https://depo-platform-documentation.scrollhelp.site/developer-docs/keyboard-testing-helper-functions)
   - Technical docs on writing good keyboard tests in Cypress -- relevant to accessibility, but should be grouped with the other Cypress docs. Currently lives under the Cypress pages, makes sense to live there.
- [Getting started with screen reader testing](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/resources/screenreader-testing-cross-platform.md)
   - GitHub documentation never brought over to Platform website. I think it could reasonably be combined with the [advanced testing / screen reader section on the "Prepare for" page](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review#Prepareforanaccessibilitystagingreview-Screenreadersscreen-readers).
- [Using VoiceOver to check for accessibility issues on MacOS](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/resources/mac-voiceover-guide.md)
   - Same as above. There's some stuff here that's specific to how VoiceOver works that we'd probably be better off linking to Apple's documentation, and the rest could be incorporated into the "Prepare for" screen reader section.

If we moved/reorganized everything as suggested, all that leaves for Accessibility under Developer Docs is [Add accessibility test cases in TestRail](https://depo-platform-documentation.scrollhelp.site/developer-docs/add-accessibility-test-cases-in-testrail) -- which actually IS about a QA tool and could live in the QA section!

I do think there's probably other accessibility documentation that would be a good fit for the Developer Docs section, it just hasn't been written yet. Most of what is here is probably only here for org chart reasons when Noah and Trevor were working on it.

So on reflection I think I'd actually be more inclined to reorganize what's here (outside the scope of this ticket) and remove the accessibility section entirely, until there's a more deliberate "accessibility documentation for engineers" initiative.

Thoughts? Does this all make sense or am I getting carried away with making changes here?

**Comment by shiragoodman** [comment]
thanks for sharing your thoughts, @briandeconinck . You definitely put more time and energy into this than I expected. 

Here's the brass tacks: we know much of the accessibility documentation under Developer Docs doesn't make sense there. But I'm also not totally convinced it all makes sense under Collab Cycle either (at least with our current format). We could even argue that some of it belongs under Design & Research too. The truth of it is, with the current hierarchal design for Platform Website, there is no nice, neat place where all of the accessibility guidance can live.

I think much of what you documented here is helpful for future decision making. However, the short and sweet intent of this ticket was to just separate the QA guidance from the Accessibility guidance, with it all still living under Developer Docs (obviously excluding the 1st two pages you linked). Brian - could I ask you to open a new ticket titled "Evaluate where Accessibility guidance should live on Platform Website" with the information in your comment in the body of the ticket? Once you do, I'm going to tag Platform Content team for assistance.

**Comment by shiragoodman** [comment]
re: this ticket, please move forward with separating QA and accessibility. If you have questions, let me know.

**Comment by briandeconinck** [comment]
Changes made in Confluence, will be published to Platform website tomorrow. Ready for review @CherylEvans!

**Comment by CherylEvans** [comment]
@briandeconinck Are all of your location changes laid out above in your comments?  If I am understanding correctly, no documents were created or changed, only their locations?

**Comment by CherylEvans** [comment]
@briandeconinck Other than that, everything looks great!

**Comment by briandeconinck** [comment]
@CherylEvans Sorry for the confusion! Per my discussion with Shira, the changes I described in the comments got punted to #50430 for further evaluation. The only changes made in this ticket are:

- Move Accessibility testing page (and child pages) out of the QA and Accessibility testing parent page
- Rename "Accessibility testing" page "Accessibility"
- Rename "QA and accessibility testing" to "Quality Assurance (QA)"

**Comment by CherylEvans** [comment]
@briandeconinck In that case, everything looks good. :-)
